experimenter/experimenter/nimbus-ui/src/lib/mocks.tsx

```diff
@@ -1141,8 +1141,19 @@ export function mockOutcomeSets(experiment: getExperiment_experimentBySlug): {
       return [];
     }
 
+    if (experiment.application === null) {
+      return slugs
+        .map((slug) => outcomes!.find((outcome) => outcome!.slug === slug))
+        .filter((outcome) => outcome != null);
+    }
```

Comment:
So you should be able to get rid of this case right?

Reply:
Yep, good catch! I forgot that this function was copied into the tests.